runtime/php/export.sh

```diff
@@ -33,3 +33,23 @@ zip --quiet --recurse-paths /export/php-${PHP_SHORT_VERSION}-fpm.zip . --exclude
 rm bootstrap
 rm bref/etc/php/conf.d/bref.ini
 rm bref/etc/php-fpm.conf
+
+# Create the PHP FPM self contained layer
+# Add files specific to this layer
+cp /layers/fpm-self/bootstrap bootstrap
+chmod 755 bootstrap
+cp /layers/fpm-self/php.ini bref/etc/php/conf.d/bref.ini
+cp /layers/fpm-self/php-fpm.conf bref/etc/php-fpm.conf
+cp /layers/fpm-self/bref/composer.json bref/composer.json
```

Comment:
Maybe we could directly copy the `composer.json` that exists at the root of the project? That would avoid to duplicate it in the layers directory? (I'm afraid it will not up be up to date at some point)

Reply:
I decided to use a minimal composer.json in order to avoid unnecessary dependencies.